qiita_db/software.py

```diff
@@ -455,6 +482,38 @@ def add_publications(self, publications):
             qdb.sql_connection.TRN.add(sql, sql_params, many=True)
             qdb.sql_connection.TRN.execute()
 
+    @property
+    def environment_script(self):
+        """The script used to start the plugin environment
+
+        Returns
+        -------
+        str
+            The script used to start the environment
+        """
+        with qdb.sql_connection.TRN:
+            sql = """SELECT environment_script
+                     FROM qiita.software
+                     WHERE software_id = %s"""
+            qdb.sql_connection.TRN.add(sql, [self.id])
+            return qdb.sql_connection.TRN.execute_fetchlast()
+
+    @property
+    def start_script(self):
+        """The start_script used to start the plugin
+
+        Returns
+        -------
+        str
+            The plugin's start script
```

Comment:
This is the one that I wanted to answer first :D
This one is executed inside the plugin environment (ie. after it has been already activated) so is the script name and we are assuming that is already in the path.